src/ixdat/readers/zilien.py

```diff
@@ -102,6 +103,22 @@ def to_mass(string):
     return None
 
 
+def determine_class(technique):
+    """Choose appropriate measurement class according to a given technique."""
```

Comment:
I think you could also just look up the technique in `ixdat.techniques.TECHNIQUE_CLASSES`. 
But I guess having this function gives you more control, e.g. with the error message for a wrong technique.

Reply:
Aha, I didn't know about that. I will update the lookup in the function. I would like to keep it exactly for the error message in order to give a nice warning.